demo/admin/src/documents/pages/blocks/TeaserItemBlock.tsx

```diff
@@ -27,6 +34,21 @@ export const TeaserItemBlock = createCompositeBlock(
                     label: <FormattedMessage id="teaserItemBlock.title" defaultMessage="Title" />,
                 }),
             },
+            htmlTag: {
+                block: createCompositeBlockSelectField<TeaserItemBlockData["htmlTag"]>({
+                    label: <FormattedMessage id="teaserItemBlock.htmlTag" defaultMessage="HTML tag" />,
```

Comment:
I am aware that we have this "technical" - `htmlTag` already in the `HeadlineBlock`, but I am not sure if a content manager is familiar with those html tags, or if they think more in Headline Levels.

Reply:
we have "HTML tag" used here https://github.com/vivid-planet/comet/blob/main/demo/admin/src/common/blocks/HeadingBlock.tsx#L74 already

Reply:
> we have "HTML tag" used here https://github.com/vivid-planet/comet/blob/main/demo/admin/src/common/blocks/HeadingBlock.tsx#L74 already

As I said, I’m aware of that.

> ...but if you say "HTML tag", "h1" would be correct, not "Headline 1"

On the other hand, if we say “Headline level”, then entries like Level 1, Level 2, Level 3 would also be correct. From an UX perspective this would, at least for me, would make more sense.